.github/workflows/validation.yaml

```diff
@@ -57,4 +57,16 @@ jobs:
         run: make test
 
       - name: run e2e example
-        run: ./hack/setup.sh cluster cartographer example-dependencies example
+        run: |-
+          sudo ./hack/ci/trust-local-registry.sh
```

Comment:
this is so that when grype tries to do the equivalent of `docker pull`, it'll not complain about the local registry being insecure (i.e., not serving content through tls w/ a cert it trusts)